Content.Server/Interaction/TilePryCommand.cs

```diff
@@ -5,19 +5,21 @@
 using Robust.Shared.Console;
 using Robust.Shared.Map;
 using Robust.Shared.Map.Components;
+using Robust.Shared.Prototypes;
 
 namespace Content.Server.Interaction;
 
 [AdminCommand(AdminFlags.Debug)]
-public sealed class TilePryCommand : IConsoleCommand
+public sealed class TilePryCommand : LocalizedEntityCommands
 {
-    [Dependency] private readonly IEntityManager _entities = default!;
+    [Dependency] private readonly ITileDefinitionManager _tileDefinitionManager = default!;
+    [Dependency] private readonly SharedMapSystem _mapSystem = default!;
 
-    public string Command => "tilepry";
-    public string Description => "Pries up all tiles in a radius around the user.";
-    public string Help => $"Usage: {Command} <radius>";
+    private readonly ProtoId<ContentTileDefinition> _platingId = "Plating";
```

Comment:
```suggestion
    private readonly string _platingId = "Plating";
```
not sure if `ContentTileDefinition` is correct here, I mixed that up
we keep it a string for now